apps/dashboard/Makefile

```diff
@@ -9,10 +32,24 @@ generate:
 		--genoperatorstate=false \
 		--noschemasinmanifest
 
+.PHONY: post-generate-cleanup
+post-generate-cleanup: ## Clean up the generated code
 	# This is a workaround for SDK codegen not producing correct output for v0alpha1
 	@rm ../../packages/grafana-schema/src/schema/dashboard/v0alpha1/types.spec.gen.ts
 	@cp ./tshack/v0alpha1_spec_gen.ts ../../packages/grafana-schema/src/schema/dashboard/v0alpha1/types.spec.gen.ts
 
 	# Same for v1alpha1
 	@rm ../../packages/grafana-schema/src/schema/dashboard/v1alpha1/types.spec.gen.ts
 	@cp ./tshack/v1alpha1_spec_gen.ts ../../packages/grafana-schema/src/schema/dashboard/v1alpha1/types.spec.gen.ts
+
+	# Remove auto-generated DeepCopy and DeepCopyInto methods for Spec for v0alpha1.
+	@sed -e '/\/\/ DeepCopy creates a full deep copy of Spec/,+5d' ./pkg/apis/dashboard/v0alpha1/dashboard_object_gen.go > ./pkg/apis/dashboard/v0alpha1/dashboard_object_gen.go.tmp
+	@sed -e '/\/\/ DeepCopyInto deep copies Spec into another Spec object/,+3d' ./pkg/apis/dashboard/v0alpha1/dashboard_object_gen.go.tmp > ./pkg/apis/dashboard/v0alpha1/dashboard_object_gen.go.tmp2
+	@rm ./pkg/apis/dashboard/v0alpha1/dashboard_object_gen.go.tmp
+	@mv ./pkg/apis/dashboard/v0alpha1/dashboard_object_gen.go.tmp2 ./pkg/apis/dashboard/v0alpha1/dashboard_object_gen.go
```

Comment:
@IfSentient this is because for v0 / v1 we do `type DashboardSpec = common.Unstructured`, which is super-extra awkward.

I guess we could have some codegen directives to control deepcopy gen, but I don't think it's worth the complexity just to cover this weird use-case.

WDYT?

Also CC @toddtreece – would love to hear your thoughts / suggestions.